### PR TITLE
Custom Toasts

### PR DESCRIPTION
**Pull request by live-wire**
This is w.r.t the issue #76 

> This PR is extended upon the contact page PR #71  (Please merge that before this)

This contains three kinds of toasts as per the design spec:
- error
- success 
- info

Please have a look @RishabhJain2018 @shivaniprakash95 @aka-jain @spyshiv

### CODE REVIEW

**Comment by live-wire** [comment]
jun-20-2018-2_56-pm
This is how the `success` toast looks like 🐝

**Comment by shivaniprakash19** [comment]
Looks Good to me!

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <!-- The <router-outlet> tells the router where to display routed views. -->
 <router-outlet></router-outlet>
+<app-toast></app-toast>
```
Comment (RishabhJain2018) [inline src/app/app.component.html:3]: Please add a new line here.

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<app-header-static></app-header-static>
+<div class="container-top container contact-container white">
+  <div class="contact-form">
+    <div class="contact-form-inner">
+      <span class="contact-title">Get in touch</span>
+      <br>
```
Comment (RishabhJain2018) [inline src/app/contact/contact.component.html:6]: I think we should avoid using `<br>` instead use the `margin-bottom`.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Comment by RishabhJain2018** [comment]
@live-wire Please update the PR.

**Comment by live-wire** [comment]
Dependent on #71 
Will resolve these after #71  is merged 🕺

**Comment by RishabhJain2018** [comment]
Closing this since it is already merged.

**Pull request closed without merging**